tox.ini

```diff
@@ -283,6 +292,8 @@ changedir=
   rgw_multisite: {toxinidir}/tests/functional/centos/7/rgw-multisite
   rgw_multisite_container: {toxinidir}/tests/functional/centos/7/rgw-multisite-container
   container_podman: {toxinidir}/tests/functional/fedora/29/container-podman
+  storage_inventory: {toxinidir}/tests/functional/centos/7/cluster
```

Comment:
I'd run a smaller scenario instead of the entire roles, for example: `lvm_osds` and `lvm_osds_container`.